app/controllers/items_controller.rb

```diff
@@ -41,16 +43,23 @@ def destroy
       render :show
     end
   end
-
   def show
   end
 
+  def get_category_children
+    @category_children = Category.find_by(id: "#{params[:parent_id]}", ancestry: nil).children
+  end
+
+  def get_category_grandchildren
+    @category_grandchildren = Category.find("#{params[:child_id]}").children
+  end
+
   private
 
   def item_params
-    params.require(:item).permit(:name, :explanation, :price, :category, :item_status_id, :postage_type_id,
-    :postage_burden_id, :shipping_area, :shipping_date_id, :trading_status_id, images_attributes: [:src, :_destroy, :id],
-    brand_attributes: [:name]).merge(user_id: current_user.id)
+    params.require(:item).permit(:name, :explanation, :price, :category_id, :item_status_id, :postage_type_id,
+    :postage_burden_id, :shipping_area_id, :shipping_date_id, :trading_status_id, images_attributes: [:src, :_destroy, :id],
```

Comment:
```suggestion
    :postage_burden_id, :shipping_area, :shipping_date_id, :trading_status_id, images_attributes: [:src, :_destroy, :id],
```
shipping_area_idから、shipping_areaに直していただければと思います！